### ISSUE TITLE
stopPing() never gets called because MessageBox.Show is blocking (modal)

### ISSUE THREAD

**Issue by ghost** [DoogeJ/NetPing#1]
![netping](https://cloud.githubusercontent.com/assets/20729073/20213732/68ac19a0-a80a-11e6-85d5-f3d196fd56e3.PNG) when trying to ping an unknown host(in this case inbox.google.nl) the program gets stuck in an infinite error loop.

Latest release, running windows 10.

**Comment by NoxiousPluK** [comment]
Hmm that's weird, it should cancel the ongoing ping. The only reason old errors should appear is if there were outstanding ping requests even after stopping.. This is a bug that I should really fix.

https://github.com/DoogeJ/NetPing/blob/master/NetPing/Form1.cs#L132

Thanks!

**Comment by ghost** [comment]
True. But it doesn't look like an old error. More as if it is stuck in a loop. If I keep it running it will just fill the screen with errors. If the issue is outstanding ping requests it  appears that the stopPing() function doesn't work correctly. 

I don't have any C# knowledge so I am just doing assumptions, hope you can fix it and awesome tool!

**Comment by NoxiousPluK** [comment]
Well I found the issue.

I should have known this, but a MessageBox() is blocking. That means that the thread that handles the ping return never actually calls stopPing() until the button on an alert is clicked and the thread continues.

To fix this I moved the stopPing() method call above the MessageBox() and also fixed the older bug that multiple alerts could pop up as outstanding pings were still returning after calling stopPing().
